lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py

```diff
@@ -67,6 +72,10 @@ def M5GAuthenticationInformation(self, request, context):
             m5gauth_vector.xres_star = m5g_ran_auth_vectors.xres_star[16:]
             m5gauth_vector.autn = m5g_ran_auth_vectors.autn
             m5gauth_vector.kseaf = m5g_ran_auth_vectors.kseaf
+            print_grpc(
```

Comment:
nit: we could also put all these `print_grpc` calls into a `finally` block, either way it's fine for me

Reply:
You are right! Didn't remember Finally was executed even after return!

will refactor that